spec/rspec/core/rake_task_spec.rb

```diff
@@ -66,6 +66,11 @@ def spec_command
         task.rspec_opts = "-Ifoo"
         expect(spec_command).to match(/#{task.rspec_path}.*-Ifoo/)
       end
+
+      it 'wont add --pattern if rspec_opts includes --pattern' do
```

Comment:
s/wont/does not/

(At least, we tend to use present-tense "does not" everywhere else in doc strings like these instead of future-tense "will not").